lxd/daemon.go

```diff
@@ -628,6 +633,10 @@ func (d *Daemon) createCmd(restAPI *mux.Router, version string, c APIEndpoint) {
 	}
 
 	route := restAPI.HandleFunc(uri, func(w http.ResponseWriter, r *http.Request) {
+		if version == "1.0" {
+			countStartedRequest(r)
```

Comment:
Can we move this to either the request or metrics package? Rather than in main in daemon.go?